testnets/026-hermippe/allocations.csv

```diff
@@ -0,0 +1,4 @@
+"amount","denom","address"
+"1_000__000_000","upenumbra","penumbrav2t1fcy6crf6u4r450k8y4nye43puxet2ytfh7s0dzxsxjk68czej9mp37xv49np0clv4dc8cwg4re0xfs79uwlfehnja4p0revmlek0drezxfse8spg3qc6gux6vyuzuuls6v6mmr"
+"100","gm","penumbrav2t1fcy6crf6u4r450k8y4nye43puxet2ytfh7s0dzxsxjk68czej9mp37xv49np0clv4dc8cwg4re0xfs79uwlfehnja4p0revmlek0drezxfse8spg3qc6gux6vyuzuuls6v6mmr"
+"1","cube","penumbrav2t1fcy6crf6u4r450k8y4nye43puxet2ytfh7s0dzxsxjk68czej9mp37xv49np0clv4dc8cwg4re0xfs79uwlfehnja4p0revmlek0drezxfse8spg3qc6gux6vyuzuuls6v6mmr"
```

Comment:
Where'd this address come from?  Is this the address for the hardcoded account used for integration tests? If not, can we add that one in?

Reply:
NVM, resolved in next commit in sequence